python/lsst/rubintv/production/rubinTv.py

```diff
@@ -1402,46 +1411,55 @@ def runM1M3HardpointAnalysis(self, event):
                                           isLiveFile=False
                                           )
 
-    def processDay(self, dayObs, skipEventNumbers={}):
+    def processDay(self, dayObs):
         """
         """
         events = self.eventMaker.getEvents(dayObs)
-        self.log.info(f'Found {len(events)} events for {dayObs=} of which'
-                      f' {len(skipEventNumbers)} have already been created')
-        mountMotionPlotted = set()
-        icsPlotted = set()
         for event in events:
-            if event.seqNum in skipEventNumbers:
-                continue
             assert event.dayObs == dayObs
+
+            nMountMotionPlots = len(self.plotsMade['MountMotionAnalysis'])
+            nM1M3HardpointPlots = len(self.plotsMade['M1M3HardpointAnalysis'])
+            # the interesting phrasing in the message is because these plots
+            # don't necessarily exist, due either to failures or M1M3 analyses
+            # only being valid for some events so this is to make it clear
+            # they've been processed.
+            self.log.info(f'Found {len(events)} events for {dayObs=} of which '
+                          f'{nMountMotionPlots} have been mount-motion plotted and '
+                          f'{nM1M3HardpointPlots} have been M1M3-hardpoint-analysed plots.')
+
+            # kind of worrying that this clear _is_ needed out here, but is
+            # _not_ needed inside each of the plotting parts... maybe either
+            # remove this or add it to the other parts?
             self.log.info(f'Plotting event {event.seqNum}')
             self.figure.clear()
             ax = self.figure.gca()
             ax.clear()
 
-            try:
-                rowData = self.runMountMotionAnalysis(event)
-            except Exception as e:
-                rowData = {event.seqNum: {'Plotting failed?': '😔'}}
-                self.log.exception(f'Failed to plot event {event.seqNum}')
-                raiseIf(self.doRaise, e, self.log)
-            finally:
-                mountMotionPlotted.add(event.seqNum)  # don't retry plotting on failure
-
-            try:
-                rowData = self.runM1M3HardpointAnalysis(event)
-            except Exception as e:
-                rowData = {event.seqNum: {'ICS processing error?': '😔'}}
-                self.log.exception(f'Failed to plot event {event.seqNum}')
-                raiseIf(self.doRaise, e, self.log)
-            finally:
-                icsPlotted.add(event.seqNum)  # don't retry plotting on failure
+            if event.seqNum not in self.plotsMade['MountMotionAnalysis']:
```

Comment:
Following from the above question, it seems like this will retry any events that already had no results, and will attempt to plot MountMotionAnalysis plots for events that did have M1M3HardpointAnalysis results and vice versa.  I think the fails are fast, and the day_obs reset will ensure things are flushed, but I just wanted to check that this is intentional.

Reply:
As discussed OOB, all is well here.